sentinel/test/unit/reminders.js

```diff
@@ -292,21 +292,17 @@ exports['sendReminder saves doc with added task to clinic'] = function(test) {
         moment: now,
         db: db
     }, function() {
-        var clinic,
-            message,
-            task;
-
         test.ok(saveDoc.called);
 
-        clinic = saveDoc.getCall(0).args[0];
+        var clinic = saveDoc.getCall(0).args[0];
```

Comment:
Is use of `var` instead of `let`/`const` deliberate?  Seems odd to see the two styles mixed here.

Reply:
No it wasn't deliberate, just doing as little refactoring as possible to keep the PR smaller. I've converted that whole test to `const`.